### PR TITLE
Remove everything

### PR DESCRIPTION
**Pull request by mvs5465**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,76 +0,0 @@
...
-    # Database configuration
-    # Defaults to using H2
-    SONARQUBE_JDBC_USERNAME=sonar \
-    SONARQUBE_JDBC_PASSWORD=sonar \
-    SONARQUBE_JDBC_URL=
-
-ENV JAVA_HOME="/usr/lib/jvm/java-1.8-openjdk"
-
-# Http port
-EXPOSE 9000
```
Comment (mvs5465) [inline Dockerfile:17]: I think we can remove `EXPOSE` as it is used only for local docker networking. 

https://we-are.bookmyshow.com/understanding-expose-in-dockerfile-266938b6a33d

File: Dockerfile [hunk 2]
```diff
@@ -1,76 +0,0 @@
...
-    SONARQUBE_HOME=/opt/sonarqube \
-    SONAR_RUNNER_VERSION=2.4 \
-    SONAR_RUNNER_HOME=/opt/sonar-runner \
-    # Database configuration
-    # Defaults to using H2
-    SONARQUBE_JDBC_USERNAME=sonar \
-    SONARQUBE_JDBC_PASSWORD=sonar \
-    SONARQUBE_JDBC_URL=
-
-ENV JAVA_HOME="/usr/lib/jvm/java-1.8-openjdk"
```
Comment (mvs5465) [inline Dockerfile:14]: Looks like Sonar 7.8 is compatible with java 11 if we want to try it out.

File: Dockerfile [hunk 3]
```diff
@@ -1,76 +0,0 @@
...
-
-# Http port
-EXPOSE 9000
-
-RUN apk update \
-    && apk upgrade \
-    && apk add --no-cache bash \
-    && apk add --no-cache --virtual=build-dependencies unzip \
-    && apk add --no-cache curl \
-    && apk add --no-cache openjdk8-jre \
```
Comment (mvs5465) [inline Dockerfile:24]: Maybe we can use an openjdk docker image instead of installing it.

File: Dockerfile [hunk 4]
```diff
@@ -1,76 +0,0 @@
...
-RUN apk add --no-cache python3 \
-    && python3 -m ensurepip \
-    && pip3 install --upgrade pip setuptools \
-    && rm -r /usr/lib/python*/ensurepip && \
-    if [ ! -e /usr/bin/pip ]; then ln -s pip3 /usr/bin/pip ; fi && \
-    if [[ ! -e /usr/bin/python ]]; then ln -sf /usr/bin/python3 /usr/bin/python; fi && \
-    rm -r /root/.cache
-
-RUN set -x \
-#   && gpg --keyserver ha.pool.sks-keyservers.net --recv-keys F1182E81C792928921DBCAB4CFCA4A29D26468DE \
```
Comment (mvs5465) [inline Dockerfile:36]: Can remove unused resources for clarity

File: ingest/ingest.py [hunk 5]
```diff
@@ -1,359 +0,0 @@
...
-        print(repo['project_name'] + ' processed')
-
-
-        break
-
-    # # # send to ES
-    print('Writing data to ES')
-    header = {'Content-type': 'application/json'}
-    es_post_response = requests.post(os.environ['elasticsearch_api_base_url'] + '/_bulk', data=document, headers=header)
-    print('Data written to ES')
```
Comment (mvs5465) [inline ingest/ingest.py:349]: Let's print the elasticsearch response here in case anything goes wrong.

File: README.md [hunk 6]
```diff
@@ -1,11 +0,0 @@
-# Codehub-Ingest (Formerly known as Hoard) ![Build Status](https://codebuild.us-east-1.amazonaws.com/badges?uuid=eyJlbmNyeXB0ZWREYXRhIjoieEg5ZDRjTUtaYkkxcTVGWFhIMEY0elFpVEhiemVsbExya2pucTJCSFZhVmFyRlFVMWNDMUF2SEFXWFhKTnYwT0NUamlBaHJxZU1WQTBOTGl6TlVXTDl3PSIsIml2UGFyYW1ldGVyU3BlYyI6ImtzODJpeUxVWnVUS0xuUEIiLCJtYXRlcmlhbFNldFNlcmlhbCI6MX0%3D&branch=master)
-
-## Required Pre-Build Configurations
-The following environment variables need to be configured:
-1. `github_user`
```
Comment (mvs5465) [inline README.md:5]: Environment variables should be UPPER_SNAKE_CASE

File: ingest/ingest.py [hunk 7]
```diff
@@ -1,359 +0,0 @@
...
-        org_repos = org_repos + ind_repo
-    return org_repos
-
-def map_repo_attributes(org_repos):
-    repos = []
-    for org_repo in org_repos:
-
-        clone_dir = cloned_project_path+'/'+org_repo['owner']['login']+'/'+org_repo['name']
-        git_url = org_repo['clone_url']
-        adjusted_url = git_url.replace('https://','https://' + os.environ['github_user'] + ':' + os.environ['github_access_token']+'@')
```
Comment (mvs5465) [inline ingest/ingest.py:41]: This should be `org_repo['owner']['login']` and not `os.environ['github_user'` I think.

File: README.md (hunk 6 above)
Comment (mvs5465) [inline README.md:5]: `github_user` is unused

**Review by mvs5465 — COMMENTED** [review]
Would recommend more modularity and OO on the python class in case we want to add to it in the future.

Looks good overall, a couple minor best practices comments.

Nice work.

File: ingest/ingest.py (hunk 7 above)
Comment (nickpricedev) [reply ingest/ingest.py:41]: No, that's for the github api authentication, not for that particular repo.

File: Dockerfile (hunk 1 above)
Comment (nickpricedev) [reply Dockerfile:17]: Good call

File: Dockerfile (hunk 2 above)
Comment (nickpricedev) [reply Dockerfile:14]: I'm good with upgrading it at a later time. Pulled most of this from existing dockerfile.

File: Dockerfile (hunk 3 above)
Comment (nickpricedev) [reply Dockerfile:24]: Also good idea for when we upgrade

File: Dockerfile (hunk 4 above)
Comment (nickpricedev) [reply Dockerfile:36]: Will do

File: ingest/ingest.py (hunk 5 above)
Comment (nickpricedev) [reply ingest/ingest.py:349]: will do

File: README.md (hunk 6 above)
Comment (nickpricedev) [reply README.md:5]: will do

Comment (nickpricedev) [reply README.md:5]: needed for github API calls

**Pull request closed without merging**